vonk/plugins/plugins_order.rst

```diff
@@ -0,0 +1,86 @@
+.. _vonk_plugins_order:
+
+The order of plugins
+====================
+
+The :ref:`VonkConfiguration attribute <vonk_reference_api_pipeline_configuration>` lets you define an ``Order``. This page explains how to choose a suitable number for that order.
+
+.. _vonk_plugins_order_inspect:
+
+Inspect order numbers in use
+----------------------------
+
+If you start Vonk, the log lists all the loaded plugins, with their order. You can see an example :ref:`here <vonk_plugins_log_pipeline>`. Also the list of :ref:`vonk_available_plugins` includes the order number chosen for each of those plugins.
+
+.. _vonk_plugins_order_minimum:
+
+Minimum order
+-------------
+
+Registering services
+^^^^^^^^^^^^^^^^^^^^
+
+If you only need to register services for the dependency injection framework, the order is only relevant if you need to override a registration done by Vonk. There are two ways:
+
+1. Choose an order before Vonk's default registration. Vonk in general uses ``TryAddSingleton`` or ``TryAddScoped`` to register an implementation of an interface. This means that if an implementation is already registered, the TryAdd... will not register a second implementation.
+
+   As an example: If you want to override the registration of ``IReadAuthorizer``: that is registered from the :ref:`RepositorySearchSupport <vonk_plugins_search>` plugin, with order 140. So you would choose an order lower than 140.
+
+2. Choose a high order (e.g. > 10000) and make sure your registration overwrites any existing registration.
```

Comment:
I didn't know about this way and it seems to involve less hassle and is less error prone than the first. Is there a reason we shouldn't just recommend this way?

Reply:
You're right